src/commands/commander.h

```diff
@@ -185,6 +187,10 @@ inline uint64_t ParseCommandFlags(const std::string &description, const std::str
       flags |= kCmdCluster;
     else if (flag == "no-dbsize-check")
       flags |= kCmdNoDBSizeCheck;
+    else if (flag == "deny-oom")
+      flags |= kCmdDenyOom;
+    else if (flag == "movable-keys")
+      flags |= kCmdMovableKeys;
```

Comment:
This flag is meaningless in Kvrocks. Please remove it and refer to KeyRangeVecGen.

Reply:
Okay, I need to learn about KeyRangeVecGen.